### PR TITLE
update trezor-connect@7

### PR DESCRIPTION
**Pull request by szymonlesisz**
Hi,
I would like to ask you to update trezor-connect version to 7.0.1
There were some major changes in ethereum protobuf messages in new Trezor firmware (1.8.0/2.1.0) and connect@6 will probably fail on "signTransaction" and "getAddress" methods.
There is also a new, required method "manifest" which needs to be added. We would like you to provide a contact to reach you in case of any required maintenance.
```javascript
Trezor.manifest({
      email: 'support@metamask.io',
      appUrl: 'https://metamask.io'
});
```

you can read more here:
https://github.com/trezor/connect/blob/develop/docs/index.md#trezor-connect-manifest

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "ethereumjs-util": "^5.1.5",
     "events": "^2.0.0",
     "hdkey": "0.8.0",
-    "trezor-connect": "^6.0.2"
+    "trezor-connect": "7.0.1"
```
Comment (brunobar79) [inline package.json]: Can you add back the `^` ?

File: index.js [hunk 2]
```diff
@@ -20,6 +20,10 @@ class TrezorKeyring extends EventEmitter {
     this.unlockedAccount = 0
     this.paths = {}
     this.deserialize(opts)
+    TrezorConnect.manifest({
+      email: 'support@metamask.io',
```
Comment (brunobar79) [inline index.js]: Would you mind moving these two as const at to the top of the file ?

**Comment by szymonlesisz** [comment]
Sure thing, it's there as requested

**Comment by brunobar79** [comment]
Thank you @szymonlesisz !

**Pull request merged**